### PR TITLE
[develop] Update base image and build script pipelines.

### PR DESCRIPTION
**Pull request by scottarnette**
## Change Description

- Add skip flags that can be set for `eosio-base-images` and `eosio-build-scripts` pipelines.
- Add Ubuntu 20.04 as a platform tested in the `eosio-base-images` pipeline.

## Change Type
**Select *ONE*:**
- [ ] Documentation

- [ ] Stability bug fix

- [ ] Other

- [ ] Other - special case

## Testing Changes
**Select *ANY* that apply:**
- [ ] New Tests

- [ ] Existing Tests

- [ ] Test Framework

- [ ] CI System

- [ ] Other

## Consensus Changes
- [ ] Consensus Changes

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

**Review by williamblevins — APPROVED** [review]

File: .cicd/base-images.yml [hunk 1]
```diff
@@ -22,6 +23,7 @@ steps:
     agents:
       queue: "automation-eks-eos-builder-fleet"
     timeout: 180
+    skip: ${SKIP_CENTOS_7}${SKIP_LINUX}
```
Comment (kj4ezj) [inline .cicd/base-images.yml:26]: The `SKIP_CENTOS_7` flag is inconsistent, albeit much better, than what we have in the [pipeline documentation](https://github.com/EOSIO/eos/blob/3950ef279b948cdaa83be3ab4e3b5355152395d8/.cicd/README.md#variables), which uses `SKIP_CENTOS_7_7`. Should these be consistent?

File: .cicd/base-images.yml [hunk 2]
```diff
@@ -56,6 +59,19 @@ steps:
     agents:
       queue: "automation-eks-eos-builder-fleet"
     timeout: 180
+    skip: ${SKIP_UBUNTU_18}${SKIP_LINUX}
```
Comment (kj4ezj) [inline .cicd/base-images.yml:62]: This is also inconsistent with our documentation:
```
SKIP_UBUNTU_16_04='true|false'       # skip all steps for Ubuntu 16.04
SKIP_UBUNTU_18_04='true|false'       # skip all steps for Ubuntu 18.04
SKIP_UBUNTU_20_04='true|false'       # skip all steps for Ubuntu 20.04
```

File: .cicd/build-scripts.yml [hunk 3]
```diff
@@ -21,6 +22,7 @@ steps:
     command:
       - "./scripts/eosio_build.sh -P -y"
     timeout: 180
+    skip: ${SKIP_CENTOS_7}${SKIP_LINUX}
```
Comment (kj4ezj) [inline .cicd/build-scripts.yml:25]: Same in this file.

File: .cicd/build-scripts.yml [hunk 4]
```diff
@@ -62,6 +65,7 @@ steps:
       - "apt update && apt upgrade -y && apt install -y git"
       - "./scripts/eosio_build.sh -P -y"
     timeout: 180
+    skip: ${SKIP_UBUNTU_18}${SKIP_LINUX}
```
Comment (kj4ezj) [inline .cicd/build-scripts.yml:68]: Ubuntus, too.

**Review by kj4ezj — CHANGES_REQUESTED** [review]
Should these be changed for consistency, or do we want to move to more human-friendly skip variable names?

**Review by kj4ezj — APPROVED** [review]
⏩

**Pull request merged**